### ISSUE TITLE
Strategy Help

### ISSUE THREAD

**Issue by combinu** [askmike/gekko#1526]
**Note: for support questions, please join our [Discord server](https://discord.gg/26wMygt)**

* **I'm submitting a ...**
[ ] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Action taken** (what you did)

`
// helpers
var _ = require('lodash');
var log = require('../core/log.js');

//var config = require('../core/util.js').getConfig();
//var settings = config.buyatsellat;

// let's create our own method
var method = {};

// prepare everything our method needs
  method.init = function() {
  this.name = 'BASA';

  this.previousAction = this.settings.init.prev_action;
  this.previousActionPrice = this.settings.init.price_action;
  this.requiredHistory = 0;
}

// What happens on every new candle?
method.update = function(candle) {
  //log.debug('in update');
}

// for debugging purposes log the last 
// calculated parameters.
method.log = function(candle) {
  //log.debug(this.previousAction)
}

method.check = function(candle) {  
  const buyat =  1 + (this.settings.tresholds.but_at / 100); // amount of percentage of difference required
  const sellat = 1 - (this.settings.tresholds.sell_at / 100); // amount of percentage of difference required
  const stop_loss_pct = 1 - (this.settings.tresholds.stop_at / 100); // amount of stop loss percentage
  const sellat_up = 1 - (this.settings.tresholds.sell_at_up / 100); // amount of percentage from last buy if market goes up

  if(this.previousAction === "buy") {
    // calculate the minimum price in order to sell
    const threshold = this.previousActionPrice * buyat;

    // calculate the stop loss price in order to sell //MP
    const stop_loss = this.previousActionPrice * stop_loss_pct;

    // we sell if the price is more than the required threshold or equals stop loss threshold
    //if((candle.close > threshold) || (candle.close < stop_loss)) {
	if((candle.close > threshold)) {	
      this.advice('short');
      this.previousAction = 'sell';
      this.previousActionPrice = candle.close;
    }
  }

  else if(this.previousAction === "sell") {
  // calculate the minimum price in order to buy
    const threshold = this.previousActionPrice * sellat;

  // calculate the price at which we should buy again if market goes up
    const sellat_up_price = this.previousActionPrice * sellat_up;

    // we buy if the price is less than the required threshold or greater than Market Up threshold
    if((candle.close < threshold) || (candle.close > sellat_up_price)) {
      this.advice('long');
      this.previousAction = 'buy';
      this.previousActionPrice = candle.close;
    }
  }
}

module.exports = method;

`

* **Expected result** (what you hoped would happen)
Seen some trading roundtrips

* **Actual result** (unexpected outcome)
I haven't seen any roundtrips on my backtest yet i get P&L percentages and also i made it against a live market and didn't trade at all after a long time. Can you please help i am fairly new and i found this code and twaeked a bit for my liking but strange i get no errors thought not working. 

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)
Using v 0.5.11 with ui 0.2.0

**Comment by askmike** [comment]
Apologies for never getting back to you, this is the bug issue tracker. See this page for better places to get help with your strategy: https://gekko.wizb.it/docs/introduction/getting_help.html
